5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
> We don't need the extra COPY and RUN statement, so it's 2 fewer layers.

It's just one fewer ;), because you need to `RUN` something. But can we optimize the layers a bit later on. On almost all other points we're following `docker-library` closely and this is a similar approach. It's really not necessary to copy and paste this into all files.